pyproject.toml

```diff
@@ -0,0 +1,5 @@
+[build-system]
+requires = ["setuptools>=42", "setuptools_scm[toml]>=3.4", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools_scm]
```

Comment:
Don't need this?
```suggestion
```

Reply:
roman and jwst both include this in this file, is it not necessary? what does including it here imply, that its needed for the setup?